salt/orchestrate/edx/services/elasticsearch.sls

```diff
@@ -25,6 +25,9 @@ generate_elasticsearch_cloud_map_file:
           - {{ salt.boto_secgroup.get_group_id(
             'salt_master-dogwood_qa', vpc_name='Dogwood QA') }}
         subnetids: {{ subnet_ids }}
+        volume_size: 200
+        tags:
+          escluster: edx-dogwood-rp
```

Comment:
Good catch, that should be qa. The reason it got added in here is because I modified the map file template to look for a `tags` variable and I am using the `escluster` tag for the micromasters cluster discovery to allow for multiple clusters in the same VPC so I'm just trying to be consistent.